### PR TITLE
Slider fix

### PR DESCRIPTION
**Pull request by dmitrijs-voronovs**
- Got rid of isVertical parameter, as it was not used anywhere
- Draggable now follows the single responsibility rule
- Fixed ProductGallery active image index

### CODE REVIEW

File: src/app/component/ProductGallery/ProductGallery.component.js [hunk 1]
```diff
@@ -68,13 +87,16 @@ class ProductGallery extends Component {
...
+        const { activeImage } = this.state;
 
         return (
             <div block="ProductGallery">
                 { this.renderAdditionalPictures(gallery) }
                 <Slider
                   mix={ { block: 'ProductGallery', elem: 'Slider' } }
                   showCrumbs
+                  activeImage={ activeImage }
+                  changeParentActiveImage={ i => this.changeActiveImage(i) }
```
Comment (xapiksw) [inline src/app/component/ProductGallery/ProductGallery.component.js]: I believe you can bind function in constructor and change this line to `changeParentActiveImage={ this.changeActiveImage }`

File: src/app/component/Slider/Slider.component.js [hunk 2]
```diff
@@ -34,8 +41,23 @@ class Slider extends Component {
         this.renderCrumb = this.renderCrumb.bind(this);
     }
 
+    static getDerivedStateFromProps(props, state) {
+        const { activeImage } = props;
+        const { prevActiveImage } = state;
+
+        if (prevActiveImage !== activeImage) {
+            return {
```
Comment (xapiksw) [inline src/app/component/Slider/Slider.component.js]: here you can write it in one line

Comment (xapiksw) [reply src/app/component/Slider/Slider.component.js]: (whole if)

Comment (alfredsgenkins) [inline src/app/component/Slider/Slider.component.js]: Let's keep it single line:
```js
return { prevActiveImage: activeImage };
```

**Pull request merged**